src/pages/myNotes/notes.jsx

```diff
@@ -1,49 +1,116 @@
+import { useLocalStorage } from '../../hooks/useLocaleStorage';
+import {
+    Backdrop,
+    ChangeDescription,
+    StyledInput,
+    Container,
+    EditButton,
+    ModalWindow,
+    NotesList,
+    StyledList,
+    StyledListComponent,
+} from './styled';
 
 const Notes = () => {
+    const [todosFromLocalStorage, setTodosFromLocalStorage] = useLocalStorage('todoList', todoList);
+
+    const [todos, setTodos] = useState([]);
     const [componentInfo, setComponentInfo] = useState({});
+    const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+    const [changeDescriptionInputValue, setChangeDescriptionInputValue] = useState('');
+
+    useEffect(() => {
+        setTodos(todosFromLocalStorage);
+    }, []);
+
+    const setDescription = (element) => {
+        setChangeDescriptionInputValue(element.target.value);
+    };
+
+    const changeDescription = () => {
+        const changedItem = todos.find((todo) => todo.id === componentInfo.id);
+        changedItem.description = changeDescriptionInputValue;
+        setTodosFromLocalStorage(todos);
+        setIsEditModalOpen(false);
+    };
+
+    const changePickedItem = (element) => {
+        getItemInfo(element);
+        setIsEditModalOpen(true);
+    };
+
+    const openModal = () => {
+        setIsEditModalOpen(true);
+    };
+    const closeModal = () => {
+        setIsEditModalOpen(false);
+    };
 
     const getItemInfo = (e) => {
-        const getCurrentItem = todoList.find((todo) => {
+        const getCurrentItem = todos.find((todo) => {
             return todo.id == e.currentTarget.id;
         });
         setComponentInfo(getCurrentItem);
+        setChangeDescriptionInputValue(getCurrentItem.description);
     };
 
     return (
-        <NotesList>
-            <Container>
-                <StyledList>
-                    {todoList.map((todo) => (
-                        <StyledListComponent
-                            isActive={componentInfo && componentInfo.id === todo.id ? true : false}
-                            key={todo.id}
-                        >
-                            <ListItemText
-                                primary={
-                                    <NoteListItem
-                                        id={todo.id}
-                                        title={todo.title}
-                                        showId={false}
-                                        description={todo.description}
-                                        date={todo.date}
-                                        getItemInfo={getItemInfo}
-                                    />
-                                }
+        <>
+            <NotesList>
+                <Container>
+                    <StyledList>
+                        {todos.map((todo) => (
+                            <StyledListComponent
+                                isActive={componentInfo && componentInfo.id === todo.id && true}
```

Comment:
&& true?

Reply:
![image](https://user-images.githubusercontent.com/73660686/146364981-7306427d-e01a-4701-ab63-8a322e1dff2a.png)
  i dont't understand. I changed, but still don't get it